### PR TITLE
gdtf: Improved creating constraints and removed pixel factor

### PR DESCRIPTION
**Pull request by nrgsille76**
- Improved creating constraints for conventional fixtures
- Removed grease pencil pixel factor wich will be removed from Blender 4.3 API

### CODE REVIEW

File: gdtf.py [hunk 1]
```diff
@@ -641,17 +641,19 @@ def get_axis(attribute):
                 check_parent = part.parent and part.parent.get("mobile_type") == "head"
                 check_pan = part.get("mobile_type") == "yoke"
                 check_tilt = part.get("mobile_type") == "head"
-                constraint = part.constraints.new('LOCKED_TRACK')
+                if not len(base.constraints) and (check_pan and not len(part.children) or (not check_pan and not check_tilt)):
+                    constraint = base.constraints.new('TRACK_TO')
+                    constraint.target = main_target
```
Comment (vanous) [inline gdtf.py:646]: getFixtureModelCollection 49: Error name 'main_target' is not defined
```
diff --git a/gdtf.py b/gdtf.py
index 6869d00..8df9938 100644
--- a/gdtf.py
+++ b/gdtf.py
@@ -643,7 +643,7 @@ class DMX_GDTF:
                 check_tilt = part.get("mobile_type") == "head"
                 if not len(base.constraints) and (check_pan and not len(part.children) or (not check_pan and not check_tilt)):
                     constraint = base.constraints.new('TRACK_TO')
-                    constraint.target = main_target
+                    constraint.target = target
```

File: gdtf.py [hunk 2]
```diff
@@ -663,7 +665,6 @@ def get_axis(attribute):
             objs["2d_symbol"] = obj
             obj.show_in_front = True
             obj.active_material.grease_pencil.show_stroke = True
-            obj.data.pixel_factor = 2
```
Comment (vanous) [inline gdtf.py:666]: What is their plan for 4.3 and up, do you know? This was to make the lines thicker...

**Review by vanous — CHANGES_REQUESTED** [review]
You can also squash your a4c99f0e2f801cff1ca4714e154ff31f92ff6052, when fixing stuff like this.

Thank you.

File: gdtf.py (hunk 2 above)
Comment (nrgsille76) [reply gdtf.py:666]: @vanous I do not know the plan, but here is a changelog for 4.3
https://docs.blender.org/api/4.3/change_log.html

**Comment by nrgsille76** [comment]
@vanous Fixed with commit 4b70a18

BTW here is the changelog for 4.3, for gpencil you can see that pixel_factor was removed
https://docs.blender.org/api/4.3/change_log.html

**Review by vanous — APPROVED** [review]

**Comment by vanous** [comment]
Thank you, squashed and merged.

**Pull request merged**